helm/examples/tracing/README.md

```diff
@@ -13,15 +13,16 @@ All configuration options detailed here are part of the HPCC Systems Helm chart,
 - exporter - Defines The type of exporter in charge of forwarding span data to target back-end
  - type - (defalt: NONE) "OTLP-HTTP" | "OTLP-GRCP" | "OS" | "NONE"
   - OTLP-HTTP
-   - url - (default localhost:4318) The url of the target OTLP-HTTP collector
+   - endpoint - (default localhost:4318) Specifies the target OTLP-HTTP backend
    - timeOutSecs - (default 10secs)
    - consoleDebug - (default false)
   - OTLP-GRCP
-   - endpoint: (default localhost:4317)     #exporter specific key/value pairs
-   - useSslCredentials
-   - sslCredentialsCACcert
+   - endpoint: (default localhost:4317) The endpoint to export to. By default the OpenTelemetry Collector's default endpoint.
+   - useSslCredentials - By default when false, uses grpc::InsecureChannelCredentials; If true uses sslCredentialsCACcert
+   - sslCredentialsCACcert - String representation of .pem file to be used for SSL encryption.
```

Comment:
This isn't secure - we will need a different way of providing he cert (via a secret) if this option is actually going to be used.  Probably best to remove this line and add support in a separate PR (please create a jira if you think it will be needed).
Alternatively add a comment that this format is deprecated and will be removed

Reply:
https://track.hpccsystems.com/browse/HPCC-30850

Reply:
alternatively for this initial offering, does it make sense to support the .pem path option instead?
HPCC-30850 will support proper k8s/vault secret